### ISSUE TITLE
Expose event in handlers for keyboard bindings

### ISSUE THREAD

**Issue by fraywing** [quilljs/quill#983]
I think this would be very helpful for situations like hotkeys where someone might want to `preventDefault` or get the original target element or something.

**Comment by jhchen** [comment]
The event object does contain more information than is passed onto the keyboard binding handlers but does present a leaky abstraction. The two particular cases mention are covered I think.

The target should always be the `.ql-editor` element and you can `preventDefault` by always called unless all handlers for a binding explicitly returns true. Are there other use cases you are running into that require the event object.

**Comment by Duder-onomy** [comment]
I have a paste handler, and I would like to get the contents of the clipboard to manually sanitize it before pasting into Quill. The quill clipboard module does not satisfy my needs in this case. 

Dealing with, and expecting events are part of being a front end developer. They should be exposed as hiding them gives us almost nothing, and exposing them gives us almost everything.

**Comment by echan00** [comment]
Curious to know what happened with this.. would be great to get original target element

**Comment by Eraldo** [comment]
Yeah, I am also facing this challenge: In my handler, I want to show a dialog at the location where the click happened, thus I would like to have the click event target available in the custom button handler.
